docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx

```diff
@@ -0,0 +1,829 @@
+ */
+async function getPlatformTransaction(transactionId) {
+  let response = await fetch(`${process.env.PLATFORM_SERVER}/transactions/${transactionId}`)
+  if (response.status != 200) {
+    throw `unexpected status code: ${response.status}`;
+  }
+  return await response.json();
+}
+
+/*
+ * Query your own database for the user based on account:memo string parameter
+ */
+function getUser(sub) {
+  return null;
+}
+
+app.listen(port, () => {
+    console.log(`business server listening on port ${port}`);
+});
+```
+
+</CodeExample>
+
+Run this with the platform server and database and initiate a new transaction with the [demo wallet][stellar-demo-wallet]. Then, we'll send the token to our server.
+
+<CodeExample>
+
+```bash
+curl \
+  -X POST \
+  -H 'Content-Type: application/json' \
+  -d '{"platformToken": "<paste the token from the URL here>"}' \
+  http://localhost:8081/session | jq
+```
+
+</CodeExample>
+
+### Providing Updates to the Platform
+
+When the user has provided you the information necessary to facilitate the transaction, you need to update the transaction's status and provide some of the information you collected to the Anchor Platform.
+
+In general, you'll want to provide updates for the following events:
+
+- Your business is ready to receive funds from the user
+- Your business has received funds from the user
+- Your business has sent funds to the user
+- Your business has processed a refund for the user's transaction
+- Your business experienced an unexpected error
+
+This is done using the Platform API's `PATCH /transactions` request. This endpoint allows you to update the data returned from the Platform API's `GET /transaction/:id` endpoint as well as the data provided to wallet applications through the applicable SEP endpoints.
+
+:::note
+
+In the next major version release of the Anchor Platform, we'll be replacing the `PATCH /transactions` Platform API endpoint with a JSON RPC endpoint that will simplify the request schema, provide better validation of data, and make the API easier to understand in general.
+
+:::
+
+Let's create an endpoint for our business server that accepts the information collected in our UI.
+
+<CodeExample>
+
+```js
+# server.js
+
+// Production systems should either let the Anchor Platform generate its own memos
+// or have your custodial service generate a memo for each transaction.
+const transactionMemos = {};
+
+app.post("/transaction", async (req, res) => {
+  let sessionToken;
+  try {
+    sessionToken = validateSessionToken(req.headers.get("authorization"));
+  } catch (err) {
+    res.status = 400;
+    res.send({ "error": err })
+    return;
+  }
+  // assuming this is a withdrawal transaction, we'll provide our distribution account as
+  // our destination account, and a memo so we can match the on-chain payment with the
+  // transaction in the Anchor Platform's database.
+  transactionMemos[req.body.transaction.id] = parseInt(Math.random() * 100000);
```

Comment:
Setting memo is optional, AP does it by itself when status is set to `pending_user_transfer_start`

Reply:
Yes, but I mention that in comments above the `transactionMemos` variable definition.

Reply:
I missed the comment, thanks